lib/at_host/at_host.c

```diff
@@ -11,28 +11,24 @@ LOG_MODULE_REGISTER(LOG_DOMAIN);
 
 #include <zephyr.h>
 #include <stdio.h>
+#include <ctype.h>
 #include <uart.h>
-#include <net/socket.h>
 #include <string.h>
 #include <init.h>
+#include <at_cmd.h>
 
-#define CONFIG_UART_0_NAME 	"UART_0"
-#define CONFIG_UART_1_NAME 	"UART_1"
-#define CONFIG_UART_2_NAME 	"UART_2"
+#define CONFIG_UART_0_NAME      "UART_0"
+#define CONFIG_UART_1_NAME      "UART_1"
+#define CONFIG_UART_2_NAME      "UART_2"
 
-#define INVALID_DESCRIPTOR 	-1
-
-#define UART_RX_BUF_SIZE 	CONFIG_AT_HOST_UART_BUF_SIZE
-
-#define THREAD_STACK_SIZE 	(CONFIG_AT_HOST_SOCKET_BUF_SIZE + 512)
-#define THREAD_PRIORITY 	K_PRIO_PREEMPT(CONFIG_AT_HOST_THREAD_PRIO)
+#define INVALID_DESCRIPTOR      -1
 
 /**
  * @brief Size of the buffer used to parse an AT command.
  * Defines the maximum number of characters of an AT command (including null
  * termination) that we can store to decode. Must be a power of two.
  */
-#define AT_MAX_CMD_LEN		CONFIG_AT_HOST_UART_BUF_SIZE
+#define AT_MAX_CMD_LEN		CONFIG_AT_CMD_RESPONSE_MAX_LEN
```

Comment:
I think you could remove the define and use the KConfig directly.
At least, could you remove the comment because it is not accurate anymore (refactoring was made in another PR).

Reply:
ok

Reply:
"Must be a power of two." Could you please remove the comment since it is not accurate anymore?

Reply:
Fixed